features/step_definitions/ontology_state_steps.rb

```diff
@@ -1,17 +1,25 @@
 Given(/^There is an ontology$/) do
   @ontology = FactoryGirl.create :ontology
-  @user = FactoryGirl.create :admin
-  login_as @user, scope: :user
+end
+
+Given(/^there is an ontology with a pending version$/) do
+  @ontology_version = FactoryGirl.create :ontology_version, state: 'pending'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.